README.md

```diff
@@ -66,42 +66,121 @@ In order to use the SDK installed via vcpkg with CMake, you can use the toolchai
 > cmake --build [build directory]
 ```
 
-The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API.
+#### Using the SDK within your Application
 
-All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality form `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
+The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API. All APIs on the client type send HTTP requests to the cloud service and return back an HTTP `Response<T>`.
+
+All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality from `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
 
 Here's an example application to help you get started:
 
 ```C++
 #include <iostream>
+
+// Include the necessary SDK headers
 #include <azure/core.hpp>
 #include <azure/storage/blobs.hpp>
 
+// Use the appropriate namespace using directives
+using namespace Azure::Storage;
 using namespace Azure::Storage::Blobs;
 
-// Get the required connection string/key from an environment variable or Azure Key Vault.
-std::string GetConnectionString();
+// Since they are secrets, get the required account name and key string from an environment variable
+// or Azure Key Vault.
+// Leaving this up to the user to implement.
+std::string GetAccountName();
+std::string GetAccountKey();
 
 int main()
 {
-  std::string containerName = "testcontainer";
-  std::string blobName = "sample-blob";
+  std::string endpointUrl = "<update to use your storage account url here>";
+  std::string accountName = GetAccountName();
+  std::string accountKey = GetAccountKey();

```

Comment:
Just a proposal: implement `GetStorageAccountUrl()`, `GetAccountName()`, `GetAccountKey()` as one-liners returning `{ return std::getenv("AZURE_STORAGE_ACCOUNT_URL"); }`, and so on.

Reply:
I sort of agree, but the sample is already a bit verbose, this is mainly getting started, and the main intention here is to highlight the basic concepts.

I would probably also expect the user to add error handling when the environment variable doesn't exist. The user has to modify the code to choose their own environment variable key name anyway.

Reply:
Alright, took your suggestion Anton, thanks!